pkg/util/trace/impl/motrace/mo_trace.go

```diff
@@ -345,6 +338,32 @@ func (s *MOSpan) doProfileRuntime(ctx context.Context, name string, debug int) {
 	}
 }
 
+func (s *MOSpan) NeedRecord() bool {
+	// if the span kind falls in mo_ctl controlled spans, we
+	// hope it ignores the long time threshold and deadline restrictions.
```

Comment:
Is there any discuss record ?

Reply:
how can make sure this logic is what you want ?

Reply:
not yet, 

but if the deadline has a higher priority than mo_ctl, the trace can not work.

Reply:
so, the key point is if `mo_ctl(..., 'SpanKindRemoteFSVis or others'， enable) ` then SpanKindRemoteFSVis‘s span MUST be record. otherwise, it should go default loop.
yes ?

Reply:
for example
If `mo_ctl(..., 'SpanKindRemoteFSVis',   DISABLE)`, the SpanKindRemoteFSVis‘s span will be
- (1) ignored,
- or (2) checked with the timeout/deadline threshold.

**Question:**
(1) or (2) ? which one is expected.

Reply:
the expected logic below:
0. if the trace is disabled, all spans do not need to be recorded.
1. if a span, controlled by mo_ctl, is enabled, it must be recorded in every invoke.
2. if a span, controlled by mo_ctl, is disabled, it must not be recorded.
3. otherwise, falls into the default loop, and then the threshold and deadline logic works.

Reply:
got it.